### PR TITLE
2022-02-21-bert_biomed_pubmed_uncased_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Comment by murat-gunay** [comment]
image

File: docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This embeddings model was imported from `Hugging Face ([link](https://huggingface.co/microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext)). It is pre trained from scratch using abstracts from PubMed and full-text articles from PubMedCentral. This model achieves state-of-the-art performance on many biomedical NLP tasks, and currently holds the top score on the Biomedical Language Understanding and Reasoning Benchmark.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md:20]: Unclosed quotes after Hugging Face

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

File: docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md:20]: @josejuanmartinez 

I will fix, but there should be a systematic problem regarding quotes. It was normal on UI

Comment (josejuanmartinez) [reply docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md:20]: @murat-gunay It's in your text, your last commits all have this problem.

**Review by josejuanmartinez — APPROVED** [review]

File: docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-02-21-bert_biomed_pubmed_uncased_en.md]: Actually I put quotes manually each time when I upload model.

**Pull request merged**